### PR TITLE
docs: Document Singularity cache managment script [FOUNDENG-333]

### PR DESCRIPTION
**Pull request by jerryharrow**
## Description

Add docs for manage-singularity-cache script.
Bump hpe-hpc-launcher version to 3.1.7 (that provides the script).

## Checklist
- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Comment by jerryharrow** [comment]
image
image

**Review by phillip-gaisford — APPROVED** [review]

File: docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst [hunk 1]
```diff
@@ -126,3 +128,55 @@ cache:
...
+
+If your system has internet access, you can download any desired docker container image (e.g.
+``determinedai/environments:py-3.8-pytorch-1.10-tf-2.8-cpu-096d730``) into the cache using the
+command:
+
+   .. code:: bash
+
+      /etc/launcher/scripts/manage-singularity-cache determinedai/environments:py-3.8-pytorch-1.10-tf-2.8-cpu-096d730
+
+Otherwise, from an internet-connected system, download the desired image using the Singularity pull
```
Comment (rcorujo) [inline docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst:167]: Does "Singularity pull" need to be enclosed in double back quotes and made lowercase?  That is, if your intent is to reference "singularity pull".

File: docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst [hunk 2]
```diff
@@ -126,3 +128,55 @@ cache:
...
+images between multiple users. It provides the following features:
+
+   -  Download the Determined default cuda, cpu, or rocm environment images
+   -  Download an arbitrary docker image reference
+   -  Copy a local Singularity image file into the cache
+   -  List the currently available images in the cache
+
+If your system has internet access, you can download images directly into the cache. Use the
+``--cuda``, ``--cpu``, or ``--rocm`` options to download the current default CUDA, CPU, or ROCM
+environment container image into the cache. For example, to load the default CUDA container image,
```
Comment (rcorujo) [inline docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst:152]: Does "to load" need to be "to download"?

**Review by rcorujo — APPROVED** [review]
Looks good. Just some minor comments.

File: docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst (hunk 1 above)
Comment (jerryharrow) [reply docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst]: Intent was not to specify the actual syntax (which is in the example).   I did change the command to ``pull`` though as that is the literal comman.

File: docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst (hunk 2 above)
Comment (jerryharrow) [reply docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst]: Changed it.  Though it does both, but more consistent to say it the same between the sentences.

**Review by rcorujo — APPROVED** [review]

**Pull request merged**